fish_prompt.fish

```diff
@@ -285,6 +285,17 @@ function __bobthefish_git_dirty_verbose -S -d 'Print a more verbose dirty state
     echo "$changes " | string replace -r '(\+0/(-0)?|/-0)' ''
 end
 
+function __bobthefish_git_stashed -S -d 'Print the stashed state for the current branch'
+    command git rev-parse --verify --quiet refs/stash >/dev/null
+    or return
+
+    echo -n "$git_stashed_glyph"
+    if [ "$theme_display_git_stashed_verbose" = 'yes' ]
+        echo -n (command git rev-list --walk-reflogs --count refs/stash 2>/dev/null)
```

Comment:
you don't need the the outer `echo -n (...)` here. it's just capturing stdout and re-echoing it :)

Reply:
You're right of course :smile: The unnecessary `echo` is removed in 40c597a.